php/data/provisioner.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * This file defines the concept of a data Provisioner.
+ *
+ * PHP Version 7
+ *
+ * @category   Data
+ * @package    Main
+ * @subpackage Data
+ * @author     Dave MacFarlane <david.macfarlane2@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+namespace LORIS\Data;
+
+/**
+ * A Provisioner is something which retrieves data from a source (usually
+ * the SQL database) and applies filters or maps to the data. It represents
+ * arbitrarily structured data such as a row in a table. Implementations
+ * know the details of the data, but a Provisioner itself only deals with
+ * Instance objects, Filters and Mappers.
+ *
+ * Filters generally do things like site based or project based permissions to
+ * the data, while mappers do things like anonymization of data.
+ *
+ * It In order to use this class, it must be extended and the child must
+ * implement the GetAllRows() function, which should try and .
```

Comment:
The row 26 should start with "In order ..." 
Row 27 "... should try and." And what exactly?

Reply:
I just took out the ", which should.." part, since it's a complete sentence on its own and the GetAllRows doc should define the requirements..